indexer/api/handlers/deposits.go

```diff
@@ -15,7 +15,8 @@ type DepositItem struct {
 	To   string `json:"to"`
 	// TODO could consider OriginTx to be more generic to handling L2 to L2 deposits
 	// this seems more clear today though
-	Tx      Transaction `json:"Block"`
+	Tx      Transaction `json:"Tx"`
+	RelayTx Transaction `json:"RelayTx"`
```

Comment:
Trying to avoid l1Tx and l2Tx because that creates some footguns on frontend for bugs compared to Tx and RelayTx

Reply:
How do you think this fits for the edge case where the deposit doesn't have an associated `RelayTx` (sends directly to the optimism portal?

Generally, is `RelayTx` needed? All information about the transfer will reside in `Tx`

Reply:
Good call should just remove for now imo. My original thought was this would be useful because we constantly get questions where someone asks where the associated relay tx hash is and it is slightly tedious to find for them

Reply:
Yea I think we should definitely include the transaction hash so that the inclusion is identifiable. The details of it is too much information